pkgs/development/python-modules/drf-pydantic/default.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  django,
+  pydantic,
+  hatchling,
+  djangorestframework,
+}:
+
+buildPythonPackage rec {
+  pname = "drf-pydantic";
+  version = "v2.7.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "georgebv";
+    repo = pname;
+    rev = version;
+    hash = "sha256-ABtSoxj/+HHq4hj4Yb6bEiyOl00TCO/9tvBzhv6afxM=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    hatchling
+  ];
+
+  propagatedBuildInputs = [
+    django
+    pydantic
+    djangorestframework
+  ];
+
```

```suggestion

  nativeChecksInputs = [
    pytestCheckHook
  ];
  
```
I think there might be some pytest flags to remove to make the tests run properly